packages/contracts-bedrock/contracts/test/SequencerFeeVault.t.sol

```diff
@@ -11,16 +12,25 @@ contract SequencerFeeVault_Test is Bridge_Initializer {
     SequencerFeeVault vault = SequencerFeeVault(payable(Predeploys.SEQUENCER_FEE_WALLET));
     address constant recipient = address(256);
 
```

Comment:
The abi change here is breaking backwards compatibility. I recommend we introduce a second event and emit the 2 events for some amount of time and then sunset the legacy event

Reply:
Agree here